schedules.py

```diff
@@ -0,0 +1,23 @@
+"""Module responsible to handle schedules."""
+import time
+from sched import scheduler
+from kytos.core.helpers import now
+from threading import Thread
+
+
+class Schedule:
+    """Schedule events."""
+
+    def __init__(self):
+        """Initialize the schedule structure."""
+        self.events = scheduler(time.time, time.sleep)
+        self.thread = None
+
+    def add_circuit_event(self, deploy_method, circuit):
```

Comment:
change method name to:

`schedule.circuit_deploy(circuit)`

`def circuit_deploy(self, circuit)`